### PR TITLE
Add "JOIN TABLE" for visits

### PR DESCRIPTION
**Pull request by kingqabil**
- Understand the different types of relationships between tables.
- Prepare complex queries that answer analytical questions.
- Use primary key & foreign key mechanisms for joining tables.

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -9,4 +9,54 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
...
+--Dean Winchester owns Angemon and Boarmon.
+UPDATE animals SET owner_id=5 WHERE name IN ('Angemon','Boarmon');
+
+-- Insert the following data for vets:
+       -- Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
+       -- Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
+       -- Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
+       -- Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.
+INSERT INTO vets(name,age,date_of_graduation) VALUES('William Tatche', 45,'2000-04-23'),
+(' Maisy Smith', 26, '2019-01-17'),('Stephanie Mendez',64,'1981-05-04'),
```
Comment (ZeenatLawal) [inline data.sql]: - [ ] There's a typo for the first name entry `William Tatcher`, kindly fix it.

File: queries.sql [hunk 2]
```diff
@@ -86,4 +86,95 @@ SELECT species, MIN(weight_kg),MAX(weight_kg) FROM animals GROUP BY species;
...
+GROUP BY owners.full_name ORDER BY total DESC LIMIT 1;
+
+/*Queries to answer the following questions asked in the project requirements: */
+
+--Who was the last animal seen by William Tatcher?
+SELECT vets.name, animals.name, date_of_visit FROM vets JOIN visits ON vets.id=visits.vets_id
+JOIN animals ON animals.id= visits.animals_id WHERE vets.name ='William Tatcher' ORDER BY visits.date_of_visit DESC LIMIT 1;
+
+--How many different animals did Stephanie Mendez see?
+SELECT COUNT(*) as total_animals from vets JOIN visits ON vets.id = visits.vets_id WHERE name='Stephanie Mendez';
```
Comment (ZeenatLawal) [inline queries.sql]: - [ ] To get the number of different animals Stephanie Mendez saw there should be a join to the animal's table also. e.g.
```sql
SELECT DISTINCT animals.name
FROM visits
LEFT JOIN animals ON animals.id = visits.animal_id
LEFT JOIN vets ON vets.id = visits.vet_id
WHERE vets.name = 'Stephanie Mendez';
```

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @kingqabil, 

Good job so far!:clap: :clap:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎊 🎊 
- Gitflow is followed. ✔️
- PR has a title and summary. ✔️


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Sadiq, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**